libs/api-bundle/src/ServiceClient/ServiceClient.php

```diff
@@ -0,0 +1,157 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\ApiBundle\ServiceClient;
+
+class ServiceClient
+{
+    private const K8S_SUFFIX = 'svc.cluster.local';
+
+    private readonly ServiceDnsType $defaultDnsType;
+
+    /**
+     * @param non-empty-string $hostnameSuffix
+     */
+    public function __construct(
+        private readonly string $hostnameSuffix,
+        ServiceDnsType|string $defaultDnsType = ServiceDnsType::PUBLIC,
```

Comment:
předpokládám, že ten string je tu kvůli symfony konfiguraci, nemohlo by to brát jenom ten string? Manuálně se to instancuje jen v testech ne?

Reply:
Upravil jsem to jeste v ty konfiguraci, aby to tam posilalo rovnou ten enum. Muzu teda ten `string` odjebat (protoze by nemelo byt potreba ho nikde rucne posilat) nebo nechat (kdyby to nahodou nekdo fakt chtel manualne vytvaret pres `services.yaml`)